src/Identity/UI/src/Areas/Identity/Pages/V3/Account/Manage/Email.cshtml.cs

```diff
@@ -172,10 +177,14 @@ public override async Task<IActionResult> OnPostSendVerificationEmailAsync()
             var userId = await _userManager.GetUserIdAsync(user);
             var email = await _userManager.GetEmailAsync(user);
             var code = await _userManager.GenerateEmailConfirmationTokenAsync(user);
+            while (code.Contains('+'))
```

Comment:
This while loop hangs

![hang](https://user-images.githubusercontent.com/1142958/61583972-cfd67480-ab37-11e9-8b9d-be7fa814754f.gif)

Reply:
Fix https://github.com/aspnet/AspNetCore/pull/12400

Reply:
Ran into this myself